py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -1,20 +1,27 @@
 #!/usr/bin/env python
 
 import argparse
-import boto3
 import json
 import sys
 
 from collections.abc import Iterable
+from typing import NamedTuple
 
 from py_gtfs_rt_ingestion import batch_files, file_list_from_s3
 
 import logging
 logging.basicConfig(level=logging.INFO)
 
-DESCRIPTION = "Generate batches of json files that should be processesd"
+DESCRIPTION = "Generate batches of json files that should be processed"
 
-def parseArgs(args) -> dict:
+class BatchArgs(NamedTuple):
+    input_file: str
+    s3_prefix: str
+    filesize_threshold: int
+    output: str
+    pretty: bool
```

Comment:
This is not necessary at all... but it does give you some nicer code completion/linting.